src/proof.rs

```diff
@@ -22,56 +22,37 @@ use poseidon252::{
     StorageScalar,
 };
 
-#[derive(Debug, Default, Clone)]
+#[derive(Debug, Clone)]
```

Comment:
Why remove the `Default` derive? Wouldn't it be useful for verifiers who need to set up the circuit?

Reply:
`PoseidonBranch` does not implement it. Also, the verifiers need to set some correct parameters that are used as PI inside of the circuit structure such as the score, prover_id or merkle_tree_root.